src/Response/JsonResponse.php

```diff
@@ -35,7 +35,7 @@ class JsonResponse extends Response
      * - JSON_HEX_AMP
      * - JSON_HEX_QUOT
      *
-     * @param string $data Data to convert to JSON.
+     * @param array $data Data to convert to JSON.
```

Comment:
Considering the internal `jsonEncode()` method typehints as `mixed`, that's what I'll do on merge.